### PR TITLE
Added jugendaemter.com

### PR DESCRIPTION
**Pull request by Jochen-A-Fuerbacher**

### CODE REVIEW

File: src/chrome/content/rules/jugendaemter.com.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+		cdn.jugendaemter.com
+-->
+<ruleset name="Jugendaemter.com">
+
+	<target host="jugendaemter.com" />
+	<target host="www.jugendaemter.com" />
+
+	<target host="unterstuetzung-die-ankommt.de" />
+	<target host="www.unterstuetzung-die-ankommt.de" />
+
```
Comment (J0WI) [inline src/chrome/content/rules/jugendaemter.com.xml]: They are not related, please split this PR.

**Comment by Bisaloo** [comment]
@J0WI, your comment has been addressed. Do you still want to work on this PR?

**Comment by J0WI** [comment]
@Bisaloo feel free to continue here. I was expecting a new ruleset for `unterstuetzung-die-ankommt.de`.

**Comment by Jochen-A-Fuerbacher** [comment]
I'am looking forward to create a new PR for `unterstuetzung-die-ankommt.de`.

File: src/chrome/content/rules/jugendaemter.com.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!--
+	Not found:
+		cdn.jugendaemter.com
```
Comment (Bisaloo) [inline src/chrome/content/rules/jugendaemter.com.xml]: This subdomain is completely gone from DNS, we can remove it.

**Comment by Bisaloo** [comment]
Also please rename this file to `Jugendaemter.com.xml` to match the `ruleset` `name`.

**Pull request merged**